diff --git a/node_modules/expo-apple-authentication/ios/AppleAuthenticationExceptions.swift b/node_modules/expo-apple-authentication/ios/AppleAuthenticationExceptions.swift
index 6eb50b1..753a6ea 100644
--- a/node_modules/expo-apple-authentication/ios/AppleAuthenticationExceptions.swift
+++ b/node_modules/expo-apple-authentication/ios/AppleAuthenticationExceptions.swift
@@ -49,6 +49,8 @@ func exceptionForAuthorizationError(_ error: ASAuthorizationError) -> Exception
   case .failed:
     return RequestFailedException()
   case .notInteractive:
     return RequestNotInteractiveException()
+  @unknown default:
+    return RequestUnknownException()
   }
 }
